CHANGELOG.md

```diff
@@ -5,6 +5,80 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.4
```

Comment:
@vieux missing date `(2016-12-06)`

which is causing build errors of rpm pkgs:
```
error: %changelog entries must start with *
```